netutils/config/parser.py

```diff
@@ -291,24 +292,24 @@ def build_config_relationship(self):
             if not line[0].isspace():
                 self._current_parents = ()
                 if self.is_banner_start(line):
-                    line = self._build_banner(line)
+                    line = self._build_banner(line)  # type: ignore
             else:
                 previous_config = self.config_lines[-1]
                 self._current_parents = (previous_config.config_line,)
                 self.indent_level = self.get_leading_space_count(line)
                 if not self.is_banner_start(line):
-                    line = self._build_nested_config(line)
+                    line = self._build_nested_config(line)  # type: ignore
                 else:
-                    line = self._build_banner(line)
+                    line = self._build_banner(line)  # type: ignore
                     if line is not None and line[0].isspace():
-                        line = self._build_nested_config(line)
+                        line = self._build_nested_config(line)  # type: ignore
                     else:
                         self._current_parents = ()
 
             if line is None:
                 break
             elif self.is_banner_start(line):
-                line = self._build_banner(line)
+                line = self._build_banner(line)  # type: ignore
```

Comment:
All these comments should be resolved by the proposed changes to the various methods.